src/core/esbuild.ts

```diff
@@ -12,10 +12,14 @@ export async function esbuildCompile(
   input: string,
   workingDir: string,
   args?: string[],
+  format?: "esm" | "iife",
 ): Promise<string | undefined> {
+  if (format === undefined) {
```

Comment:
Can we go with `format = format ?? "esm"`? we've been bit with falsey issues before.

Reply:
FYI I'm doing a pass that changes all of the "if (x === undefined) { x = ... }" I find in my source to `x = x ?? ...`.

Reply:
Would `format = format ?? "esm"` catch empty string? (I think we'd want to default empty string to "esm" so perhaps falsey is okay here? Otherwise I'm generally quite in favor of `??`